### ISSUE TITLE
No Device found

### ISSUE THREAD

**Issue by brucetoo** [pbreault/adb-idea#59]
Android studio 2.2.2
Once i use Adb idea in one project, when i change to the other project,
Adb idea always alert "No Device found". Although i uninstall this plugin 
and reinstall it.

**Comment by pbreault** [comment]
Is it one project in particular that is broken or are you saying that if you use it in one project, the other projects that are currently running are not working?

Are the devices visible from `adb devices` from the command line and from the run dialog in android studio ?

**Comment by brucetoo** [comment]
Yes, if i use it in one project, the other projects that are currently running are not working.

`adb devices` command can show the connected devices, and other adb commands work well.

**Comment by pbreault** [comment]
And is it always the same project that works and the other never work or is it whatever project you opened first that wins ?

What gradle plugin version are you using in these projects ?

**Comment by brucetoo** [comment]
It's the first opened project wins, and not work for any other project.
my local plugin is ` 1.4.0` the latest one. BTW, the command like `adb install/uninstall package_name` still works for any projects.

**Comment by pbreault** [comment]
I can't reproduce this on android 2.2.3 and on 2.3.0-beta4

Can you test on these versions to see if it fixes anything?
And is there an error message / stacktrace in the event log ?

![image](https://cloud.githubusercontent.com/assets/279299/22980175/bfdc5eae-f366-11e6-893e-9d094d4f1271.png)

**Comment by brucetoo** [comment]
I update AS up to 2.2.3, and test, still only works on the first won project . Even i uninstall plugin and install again. Here are event log.
<img width="696" alt="screen shot 2017-02-16 at 09 24 15" src="https://cloud.githubusercontent.com/assets/6925516/23003186/f6847bda-f429-11e6-9223-22b64dec2bc7.png">

**Comment by ycdm** [comment]
AS2.3 Not work
alert ：
ADB IDEA: No Device found

**Comment by ckesc** [comment]
@brucetoo check out new 1.4.1 version! Seems it fixed bug.

**Comment by brucetoo** [comment]
Android studio plugins platform is still old version..
When i install plugin from disk, Here is error tips..so weird!
![adb](https://cloud.githubusercontent.com/assets/6925516/23993081/2a31187e-0a7a-11e7-8f18-2e5ac05ead0e.png)

**Comment by pbreault** [comment]
1.4.1 is a hotfix for Android studio 2.4 , it can't be installed on 2.3 and it doesn't fix the same bug.
The error message is the same mainly because it's a generic error that gets handled in a top-level try/catch.

I was able to reproduce this one recently, I'll give it a shot.

**Comment by Nemisis** [comment]
Note: This was happening to me in Android Studio 3.0 for a while. I uninstalled the plugin, restarted Android Studio, reinstalled it and restarted Android Studio again. It works now.

**Comment by pcqpcq** [comment]
reinstall works for me, looks like this is caused with upgrade.

**Comment by arpit07531** [comment]
its still not working for me. Whenever I start a new android studio instance, adb-idea stops working in the previously opened window. It shows this error - ADB IDEA: No Device found
It never works in multiple android studio windows.
